src/commands/vote.ts

```diff
@@ -0,0 +1,18 @@
+import { FireMessage } from "../../lib/extensions/message";
+import { Language } from "../../lib/util/language";
+import { Command } from "../../lib/util/command";
+
+export default class extends Command {
+  constructor() {
+    super("vote", {
+      description: (language: Language) =>
+        language.get("VOTE_COMMAND_DESCRIPTION"),
+      clientPermissions: ["SEND_MESSAGES"],
+      aliases: ["dbl", "top.gg", "dboats", "discord.boats"],
+    });
+  }
+
+  async exec(message: FireMessage, args: { name: string }) {
```

Comment:
```suggestion
  async exec(message: FireMessage) {
```
There are no args in the command options so the exec method shouldn't take args

Reply:
ok, will remember that